dds/DCPS/TimeTypes.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * Monotonic Time Backport for 3.13. Just provides the bare minimum needed for
+ * monotonic time.
+ */
+
+#ifndef OPENDDS_DCPS_TIME_TYPES_HEADER
+#define OPENDDS_DCPS_TIME_TYPES_HEADER
+
+#include "ace/Monotonic_Time_Policy.h"
+#include "ace/Condition_Attributes.h"
+#include "ace/Time_Policy.h"
+
+#if !defined (ACE_LACKS_PRAGMA_ONCE)
+#  pragma once
+#endif /* ACE_LACKS_PRAGMA_ONCE */
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+/**
+ * ACE_Time_Policy that OpenDDS uses for internal timing.
+ *
+ * ACE_Monotonic_Time_Policy protects OpenDDS from being effected by changes to
+ * the system clock to a certain degree.
+ */
+///@{
+#if defined(__APPLE__) && defined(__MACH__)
+/*
+ * As of writing, ACE_Monotonic_Time_Policy doesn't support Darwin systems like
+ * macOS. Use ACE_System_Time_Policy instead, because ACE_Monotonic_Time_Policy
+ * falls back to retuning ACE_Time_Value::zero for some reason.
```

Comment:
```suggestion
 * falls back to returning ACE_Time_Value::zero for some reason.
```

Reply:
I spell checked earlier that word but must have hit the wrong suggestion....